src/org/usfirst/frc/team2585/systems/WheelSystem.java

```diff
@@ -0,0 +1,94 @@
+package org.usfirst.frc.team2585.systems;
+
+import org.usfirst.frc.team2585.Environment;
+import org.usfirst.frc.team2585.RobotMap;
+import org.usfirst.frc.team2585.input.InputMethod;
+
+import edu.wpi.first.wpilibj.RobotDrive;
+import edu.wpi.first.wpilibj.Victor;
+
+public class WheelSystem implements RobotSystem, Runnable {
+	
+	private RobotDrive drivebase;
+	private InputMethod input;
+	
+	private Victor rightUpperDrive;
```

Comment:
It might be good to just declare these as the super class SpeedController and initialize them as Victors in the init function. That way if we switch the type of motor controller then we just need to change it in one place.